app/routes/surveys/components/surveys.css

```diff
@@ -191,8 +191,11 @@
 }
 
 .templateDropdown {
-  position: absolute;
   color: transparent;
+  position: absolute;
+  bottom: 0;
+  height: 0;
+  width: 0;
```

Comment:
Wouldn't hurt with a comment but okay

Reply:
It wouldn't hurt, but I'm planning on refactoring it soon™️